### PR TITLE
updates locale order and adds some tests

### PR DESCRIPTION
**Pull request by tjbenton**

### CODE REVIEW

File: utils/locale/test/locale.test.mjs [hunk 1]
```diff
@@ -2,25 +2,45 @@ import { LocaleManager } from '../src/locale'
...
+    expect(l.parse('USD')).toBe(obj)
+  })
+
+  test('is', () => {
+    l.setLocale('en-US')
+
+    expect(l.is('en-US')).toBeTruthy()
+    expect(l.is('United States')).toBeTruthy()
+    expect(l.is('en')).toBeTruthy()
+    expect(l.is('US')).toBeTruthy()
```
Comment (alex-andreescu) [inline utils/locale/test/locale.test.mjs:26]: This line is the same as the one below

File: utils/locale/test/locale.test.mjs [hunk 2]
```diff
@@ -2,25 +2,45 @@ import { LocaleManager } from '../src/locale'
...
+    expect(l.is('US')).toBeTruthy()
+    expect(l.is('USA')).toBeTruthy()
+    expect(l.is('USD')).toBeTruthy()
+    expect(l.is('en')).toBeTruthy()
```
Comment (alex-andreescu) [inline utils/locale/test/locale.test.mjs]: This is the same as line 25

File: utils/locale/src/locale.mjs [hunk 3]
```diff
@@ -14,6 +14,7 @@ const find = (values, _, omittedValues = []) => {
 
 export class LocaleManager {
   static localeItems = [
```
Comment (mmurtagh) [inline utils/locale/src/locale.mjs]: Not a huge deal, but this is an enormous data structure defined inline with code. I feel like when it gets to this point it warrants its own `.json` file and let it be imported. This just makes this class hard to read and distracts from the important logic bits. Maybe we can write a comment here outlining the basic structure instead?

Comment (tjbenton) [reply utils/locale/src/locale.mjs]: done

**Pull request merged**